resources/graphClient/graphClient.html

```diff
@@ -10,19 +10,21 @@
     This is the HTML for exploring graphs, and is hosted inside a WebView in a VS HTML preview window
 -->
 
```

Comment:
You might want to create a VSIX and test it with the VSIX to make sure webpack is working with your changes.

Reply:
Done. :-)